corehq/apps/es/tests/test_client.py

```diff
@@ -418,6 +470,29 @@ def test_index_refresh(self):
             self.adapter.index_refresh(self.index)
             patched.assert_called_once_with([self.index])
 
+    def test_reindex(self):
```

Comment:
It looks like this tests the "happy path" for one set of arguments (that is, `wait_for_completion=True, refresh=True`). The `refresh` kwarg is ubiquitous across the Elastic API, so that probably doesn't need special tests, but if the manager's `reindex()` method needs to expose the `wait_for_completion` arg, then there should be a test for the `wait_for_completion=False` condition. Possibly also the behavior of the `request_timeout` kwarg.

Reply:
:medal_sports: